torch/optim/nadam.py

```diff
@@ -54,8 +56,17 @@ def _init_group(self, group, params_with_grad, grads, exp_avgs, exp_avg_sqs, mu_
                 state = self.state[p]
                 # Lazy state initialization
                 if len(state) == 0:
-                    state['step'] = torch.tensor(0.)
-                    state['mu_product'] = torch.tensor(1.)
+                    # note(crcrpar): [special device hosting for step]
```

Comment:
is `note(crcrpar)` typo or something?

Reply:
no it's copied from here: https://github.com/pytorch/pytorch/blob/e35cb480f4df1cf440b8705c93546c1b15891a4b/torch/optim/adam.py#L88C1-L90C82

The same reasoning applies here

Reply:
haha we can remove the note if you want your authorship to not be attached there @crcrpar

Reply:
was wondering if sth like `see note - [special device hosting for step]` happens to be a little bit clearer. I just felt it a bit surprising to see my ID as a comment author while the comment itself is "author"ed by someone else